ceph.spec.in

```diff
@@ -1381,7 +1381,12 @@ fi
 %{_bindir}/ceph-monstore-tool
 %{_mandir}/man8/ceph-mon.8*
 %{_mandir}/man8/ceph-rest-api.8*
+%if 0%{with python2}
 %{python_sitelib}/ceph_rest_api.py*
+%else
+%{python3_sitelib}/ceph_rest_api.py*
+%{python3_sitelib}/__pycache__/ceph_rest_api.cpython*.py*
```

Comment:
nit, could be more specific:
```
%{python3_sitelib}/__pycache__/ceph_rest_api.cpython*.py[co]
```

Reply:
This same pattern occurs throughout the spec file. The `*` is pretty standard. Although I agree that `[co]` form would be more specific, I'm not sure if RPM supports it. In short, I'd rather leave this line as it is.